application/src/main/resources/ecc.yml

```diff
@@ -87,6 +87,13 @@ connection:
     ## Extending this allows to manipulate the SSLEngine and SSLParameters.
     ##
     certificateHandler: com.ericsson.bss.cassandra.ecchronos.application.config.security.ReloadingCertificateHandler
+    ##
```

Comment:
Fix indentation, on the same level of the property

Reply:
Done